appservice/src/intTest/groovy/nu/yona/server/CreateUserOnBuddyRequestTest.groovy

```diff
@@ -347,7 +347,9 @@ class CreateUserOnBuddyRequestTest extends AbstractAppServiceIntegrationTest
 				],
 				"goals":[
 					{
-						"activityCategoryName":"news"
+						"@class":"budgetGoal",
```

Comment:
Agreed. It took me most time to implement the polymorphism like this but I realized it was lost time anyway :(. Maybe it is good to do YD-165 first or in combination with adding other goal types to take the impact as early as possible.